src/administration/integrations/blackfire.md

```diff
@@ -39,7 +39,7 @@ git push
 
 ### 3. Confirm it's running
 
-You can then confirm that Blackfire is running by SSH in the PHP container:
+You can then confirm that Blackfire is running from SSH in the PHP container as follows:
```

Comment:
Both versions here seem clumsy.  I think it's intending to say "you should SSH into the container and run this command."  Let's just rewrite the whole sentence to avoid using "SSH" as a verb or using the clumsy "SSHing". :smile: